File: backend/src/main/java/com/ftn/activityapp/dto/RegisterUserRequest.java
package com.ftn.activityapp.dto;

import com.ftn.activityapp.enums.ActivityLevel;
import com.ftn.activityapp.enums.UserGoal;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class RegisterUserRequest {
    private String email;
    private String password;
    private String firstName;
    private String lastName;
    private Integer age;
    private Double height;
    private Double weight;
    private ActivityLevel activityLevel;
    private UserGoal goal;
    private Integer goalSteps;
}

File: mobile-app/app/src/main/java/com/example/activityapp/data/remote/dto/RegisterUserRequest.kt
package com.example.activityapp.data.remote.dto

data class RegisterUserRequest(
    val email: String,
    val password: String,
    val firstName: String,
    val lastName: String,
    val age: Int,
    val height: Double,
    val weight: Double,
    val activityLevel: String,
    val goal: String,
    val goalSteps: Int
)